### PR TITLE
[New] Converted test cases for action dialog components from enzyme t…

### PR DESCRIPTION
**Pull request by gitdivyanshu**
…o rtl

### CODE REVIEW

File: app/javascript/components/action-dialog/component.spec.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    };
+    mountedComponent(<ActionDialog {...newProps} />);
+    expect(screen.getAllByRole('button', { className: 'MuiSvgIcon-root' })).toBeTruthy();
+  });
+
+  it("should render cancel Button", () => {
+    expect(screen.queryByText("cancel")).toBeInTheDocument();
+  });
+
+  it('should accept valid props', () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/action-dialog/component.spec.js:50]: Remove this test

File: app/javascript/components/action-dialog/component.spec.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    expect(screen.getByRole("dialog")).not.toHaveClass('dialogSubtitle');
+  });
+
+  it("should not render dialogSubHeader because isn't passed in props ", () => {
+    delete props.dialogSubHeader;
+    mountedComponent(<ActionDialog {...props} />);
+    expect(screen.getByRole("dialog")).not.toHaveClass('dialogSubHeader');
+  });
+
+  // it("should render DialogActions", () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/action-dialog/component.spec.js:89]: Did you have trouble converting this test?

File: app/javascript/components/action-dialog/components/text-with-close/component.spec.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    const newProps = {
+      ...props,
+      className: "MuiSvgIcon-root"
+    };
+    mountedComponent(<TitleWithClose {...newProps} />);
+    expect(screen.getAllByRole('button', { className: 'MuiSvgIcon-root' })).toBeTruthy();
+  });
+
+  it('should accept valid props', () => {
+    const newProps = {
```
Comment (jtoliver-quoin) [inline app/javascript/components/action-dialog/components/text-with-close/component.spec.js:34]: Remove this test

**Review by jtoliver-quoin — APPROVED** [review]

File: app/javascript/components/action-dialog/component.spec.js (hunk 1 above)
Comment (gitdivyanshu) [reply app/javascript/components/action-dialog/component.spec.js]: done

File: app/javascript/components/action-dialog/component.spec.js (hunk 2 above)
Comment (gitdivyanshu) [reply app/javascript/components/action-dialog/component.spec.js]: yes i have make this test case todo

File: app/javascript/components/action-dialog/components/text-with-close/component.spec.js (hunk 3 above)
Comment (gitdivyanshu) [reply app/javascript/components/action-dialog/components/text-with-close/component.spec.js]: done

**Pull request merged**